include/tvm/relay/module.h

```diff
@@ -102,6 +109,59 @@ class ModuleNode : public RelayNode {
    */
   void Update(const Module& other);
 
+  /*!
+   * \brief Get the entry point of the module.
+   *
+   * \returns The entry point function, (i.e. main).
+   */
+  Expr EntryPoint();
```

Comment:
I am a bit confused by the difference between entry_func and EntryPoint

Reply:
We can name it `EntryFunc` if that makes it clearer, I stated my reasoning for using a global var below.